tests/php/Extension/FluentVersionedExtensionTest/TestVersionedModel.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace TractorCow\Fluent\Tests\Extension\FluentVersionedExtensionTest;
+
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Versioned\Versioned;
+use TractorCow\Fluent\Extension\FluentVersionedExtension;
+
+/**
+ * @mixin Versioned
+ */
+class TestVersionedModel extends DataObject implements TestOnly
```

Comment:
I think your test model should explicitly define the locale inheritance mode rather than relying on default to prevent ambiguity.

Reply:
Can you please provide me with an appropriate snippet for that? I'm not sure which locale inheritance mode this should use if declared explicitly.